test/integration/tsconfig.json

```diff
@@ -1,9 +1,11 @@
 {
-    "compilerOptions": {
-      "resolveJsonModule": true,
-      "lib": ["es6", "es7"],
-      "noUnusedLocals": true,
-      "esModuleInterop": true,
-      "allowSyntheticDefaultImports": true
-    }
-  }
\ No newline at end of file
+  "compilerOptions": {
+    "resolveJsonModule": true,
+    "lib": ["es6", "es7"],
+    "module": "CommonJS",
+    "target": "ES5",
+    "noUnusedLocals": true,
```

Comment:
you were a bit strict here 😄  it now return errors on issues that are most likely warning for eg:
Error: test/integration/ComputeFlow.test.ts(1,1): error TS6133: 'config' is declared but its value is never read.
but we can handle those and delete unused vars

Reply:
yeah, figured there's no need for all those individual configs and merged them all into one place in fdcf7df. And made `"noUnusedLocals": false` so I hope now everything gets green again.